src/apollo/resolvers/user.ts

```diff
@@ -11,7 +11,14 @@ import Messages from "../../utils/messages";
 
 const saveProfilePicture = async (user: any, image: any) => {
   if (image) {
-    const path = await saveImage(image);
+    let path;
+
+    try {
+      path = await saveImage(image);
+    } catch (e) {
+      throw new ApolloError(" Unable to upload image(s)\nError response: " + e);
```

Comment:
Can we move this to where the `return e;` statement currently is in `signUp`? I'm thinking we can get the same error output by only having the try catch and throw in `signUp`.